build.sbt

```diff
@@ -233,26 +232,15 @@ lazy val benchmarks = project
 lazy val docs = project
   .in(file("zio-prelude-docs"))
   .settings(
-    scalaVersion                               := BuildHelper.Scala213,
-    publish / skip                             := true,
-    moduleName                                 := "zio-prelude-docs",
+    scalaVersion   := BuildHelper.Scala213,
+    publish / skip := true,
+    moduleName     := "zio-prelude-docs",
     scalacOptions -= "-Yno-imports",
-    scalacOptions -= "-Xfatal-warnings",
-    ScalaUnidoc / unidoc / unidocProjectFilter := inProjects(
-      core.jvm,
-      experimental.jvm,
-      experimentalLaws.jvm,
-      laws.jvm,
-      scalaParallelCollections
-    ),
-    ScalaUnidoc / unidoc / target              := (LocalRootProject / baseDirectory).value / "website" / "static" / "api",
-    cleanFiles += (ScalaUnidoc / unidoc / target).value,
-    docusaurusCreateSite                       := docusaurusCreateSite.dependsOn(Compile / unidoc).value,
-    docusaurusPublishGhpages                   := docusaurusPublishGhpages.dependsOn(Compile / unidoc).value
+    scalacOptions -= "-Xfatal-warnings"
   )
   .settings(macroDefinitionSettings)
   .dependsOn(core.jvm, experimental.jvm, experimentalLaws.jvm, laws.jvm, scalaParallelCollections)
-  .enablePlugins(MdocPlugin, DocusaurusPlugin, ScalaUnidocPlugin)
+  .enablePlugins(WebsitePlugin)
```

Comment:
Does WebsitePlugin do the unidoc stuff?

Reply:
Not right now, but I will add that to the subsequent iterations. Currently, the goal is to integrate all docs into one place. Then I will focus on AP‌I docs.

Reply:
Fair enouch